client/app/domain/zone/record/resume.html

```diff
@@ -19,30 +19,23 @@
         <dd class="word-break" data-ng-bind="ctrl.model.target.value"></dd>
     </dl>
 
-    <div class="alert alert-warning mt-4" role="alert"
-         data-ng-if="ctrl.getResumeTargetAlert()">
-        <div data-ng-bind-html="'domain_configuration_dns_entry_add_step3_warning' | translate: { t0: ctrl.getResumeTargetAlert() }"></div>
-    </div>
-
     <div class="alert alert-danger mt-4" role="alert"
          data-ng-if="!ctrl.recordConflicts">
-        <div>
-            <span data-translate="domain_configuration_dns_entry_add_step3_check_warning1"></span>
-
-            <ul data-ng-if="ctrl.model.fieldType === 'CNAME'">
-                <li data-translate="domain_configuration_dns_entry_add_step3_check_warning2"></li>
-                <li data-translate="domain_configuration_dns_entry_add_step3_check_warning3"></li>
-            </ul>
-            <ul data-ng-if="ctrl.model.fieldType === 'A' || ctrl.model.fieldType === 'AAAA'">
-                <li data-translate="domain_configuration_dns_entry_add_step3_check_warning4"></li>
-            </ul>
-            <ul data-ng-if="ctrl.model.fieldType !== 'CNAME' && ctrl.model.fieldType !== 'A' && ctrl.model.fieldType !== 'AAAA'">
-                <li data-translate="domain_configuration_dns_entry_add_step3_check_warning5"></li>
-            </ul>
+        <div data-ng-if="ctrl.model.fieldType === 'CNAME'"
+             data-translate="domain_configuration_dns_entry_add_step3_check_warning2">
+        </div>
+        <div data-ng-if="ctrl.model.fieldType !== 'CNAME'"
+             data-translate="domain_configuration_dns_entry_add_step3_check_warning3">
         </div>
     </div>
 
+    <div class="alert alert-warning mt-4" role="alert"
```

Comment:
Could be turned later into a [`<oui-message>`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/message) element:)